Toolkit/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -174,8 +174,11 @@ public class MeasureToolbar: UIToolbar, AGSGeoViewTouchDelegate {
     // Exposed so that the symbology and selection colors can be customized.
     public private(set) var selectionLineSymbol : AGSSymbol?
     public private(set)var selectionFillSymbol : AGSSymbol?
-    public private(set) var selectionColor : UIColor?
-    
+    @available(iOS, deprecated, message: "Use `color` property exposed through `AGSGeoView#selectionProperties`")
```

Comment:
![screen shot 2018-10-11 at 1 40 31 pm](https://user-images.githubusercontent.com/2257493/46832779-cb389080-cd5b-11e8-92d8-2829bb6a3dd7.png)

Does the message get parsed for documentation? If not, I would suggest not using `AGSGeoView#selectionProperties` as it is a non-standard symbol name. The Swift symbol name is `AGSGeoView.selectionProperties`. You could use that, or you might consider using the key path:

![screen shot 2018-10-11 at 1 42 09 pm](https://user-images.githubusercontent.com/2257493/46832839-f91dd500-cd5b-11e8-81bf-fd1dfbe00743.png)

Reply:
You also might consider moving the property into an extension so that it doesn't clutter up the class implementation.

Reply:
I added the `.` instead of `#`, but left the form of the message the same, since it matches the SDK API's message.

Thanks.